.github/workflows/conda-build-test.yml

```diff
@@ -0,0 +1,29 @@
+name: conda builds
+
+on:
+  pull_request:
+    paths:
+      - 'conda-environment.yml'
+  schedule:
+    - cron: '17 22 * * 6'
+  workflow_dispatch:
+
+jobs:
+  test_conda_install:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: checkout cylc-flow
+        uses: actions/checkout@v2
+
+      - name: build conda env
+        run: |
+            conda env create \
+              -f conda-environment.yml
+
+      - name: check cylc-version
+        run: |
+          find /usr/share/miniconda/envs/cylc-dev/. -name "activate.d" | tee > activates.txt
```

Comment:
@oliver-sanders Do you know of any packages which use activate scripts?
I have struggled to web-search such a thing, and therefore to check this test against a control.

Reply:
Here's one:

https://github.com/conda-forge/cupy-feedstock/blob/5f118fca132164c0d0699d508e45bdf18fba4f5a/recipe/activate.sh#L4-L6

Which is copied into the `activate.d` directory in the build:

https://github.com/conda-forge/cupy-feedstock/blob/5f118fca132164c0d0699d508e45bdf18fba4f5a/recipe/meta.yaml#L49-L52